src/Auth/AccessTokenCacheProvider.php

```diff
@@ -26,7 +26,7 @@ public function __construct(IHttpClient $client, IStorage $storage)
 		$this->cache = new Cache($storage, self::CACHE_NAMESPACE);
 	}
 
-	public function getAccessToken(Config $config): AccessToken
```

Comment:
Hold on, it's a BC break. Why is that?

Reply:
In this method is possible that return statement is NULL, but the return type wasn't nullable and PHPStan was sad.

Reply:
Hmm, we can't change a notation. Better to throw an exception or sth like that.